### ISSUE TITLE
Allow new M series FIN

### ISSUE THREAD

**Issue by zawmyolatt** [GovTechSG/singpass-myinfo-oidc-helper#34]
Getting this error
```
Token payload sub property is invalid, does not contain valid NRIC and uuid string
```
when call `getIdTokenPayload` function after `M` series user login.

***Propose Solution***
- fix this reg expression `/s=([STFG]\d{7}[A-Z]).*,u=(.*)/i`  to `/s=([STFGM]\d{7}[A-Z]).*,u=(.*)/i` allow `M` series 

https://github.com/GovTechSG/singpass-myinfo-oidc-helper/blob/5b9af98ece241eb377d6b5af5423703e5b9a1b06/src/singpass/singpass-helper.ts#L190
https://github.com/GovTechSG/singpass-myinfo-oidc-helper/blob/5b9af98ece241eb377d6b5af5423703e5b9a1b06/src/util/RedactorUtil.ts#L9

**Comment by darren0608** [comment]
@yiyangtan kindly assist asap, as this is preventing FIN users with M series from getting their endorsed medical certificate for outbound travel, thanks

**Comment by dyihoon90** [comment]
the team is working on updating the code. In future for time-critical requirements you all can fork the repo and make the changes in the fork first. Even better to contribute back with a pull request :)

**Comment by Gyunikuchan** [comment]
The fix has been published via v6.1.11
